app/(drawer)/(tabs)/send.tsx

```diff
@@ -63,10 +63,10 @@ export default function Send() {
     <Frame>
       <HeaderBar title="Send" pillContent={sendPillContent} />
       <View className="flex-1 gap-4">
-        <View>
+        <View className="mb-4 flex-1">
```

Comment:
This is unnecessary, the parent View already handles the margins between the children
```suggestion
        <View>
```

Reply:
it was a previous issue where the view scroll was messing things up but not one anymore yes